plugins/package-managers/go/src/main/kotlin/GoMod.kt

```diff
@@ -454,7 +455,8 @@ private fun ModuleInfo.toSourceArtifact(): RemoteArtifact {
 }
 
 private fun ModuleInfo.toVcsInfo(): VcsInfo? {
-    val infoFile = goMod?.let { File(it).resolveSibling("$version.info") } ?: return null
+    val escapedVersion = escapeVersion(version)
```

Comment:
To increase readability I believe we should define an extension property:

```
var ModuleInfo.escapedVersion: String
    get() = escapeVersion(version)
```

Reply:
Kept as it is. This Kotlin Woodoo confuses me. I think the readabliity is already good.